### PR TITLE
Add view users API

### PR DESCRIPTION
**Pull request by SomTambe**
Added the view users API as said by nilay
Merge at the earliest.

### CODE REVIEW

**Comment by SomTambe** [comment]
API has been tested and is perfectly working

File: README.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 ## API Endpoints
 A description of all the API endpoints, their URL and request parameters.
 ### Users
+#### View User
+```
+url : /users/viewuser
+method : GET
+
+Successful : 200_OK
```
Comment (namangup) [inline README.md:10]: Add the response format

File: README.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
 ## API Endpoints
 A description of all the API endpoints, their URL and request parameters.
 ### Users
+#### View User
+```
+url : /users/viewuser
```
Comment (namangup) [inline README.md]: Change the url to something like /users/profile/
Not the forward slash at the end.

File: users/views.py [hunk 3]
```diff
@@ -17,7 +17,17 @@
...
-#@csrf_exempt
+from users.serializers import UserViewSerializer
+# @csrf_exempt
+
+class UserView(APIView):
+
+    def get(self, request):
+        user = IsLoggedIn(request)
+        serializer = UserViewSerializer(user)
+        if user is None:
```
Comment (namangup) [inline users/views.py]: Use 401_UNAUTHORIZED

Check the user before serializing the user object

File: users/views.py [hunk 4]
```diff
@@ -17,7 +17,17 @@
...
 from rest_framework.renderers import JSONRenderer, TemplateHTMLRenderer
 from bookmark.models import Bookmark
-#@csrf_exempt
+from users.serializers import UserViewSerializer
+# @csrf_exempt
+
+class UserView(APIView):
+
+    def get(self, request):
+        user = IsLoggedIn(request)
```
Comment (namangup) [inline users/views.py]: Use a try except block to gracefully return exceptions

File: README.md (hunk 1 above)
Comment (SomTambe) [reply README.md:10]: Done

File: README.md (hunk 2 above)
Comment (SomTambe) [reply README.md]: Done

File: users/views.py (hunk 3 above)
Comment (SomTambe) [reply users/views.py]: Done

File: users/views.py (hunk 4 above)
Comment (SomTambe) [reply users/views.py]: Done

**Review by namangup — APPROVED** [review]

**Pull request merged**